src/ILLink.Tasks/LinkTask.cs

```diff
@@ -132,30 +124,31 @@ protected override string GenerateCommandLineCommands ()
 		{
 			var args = new StringBuilder ();
 			args.Append (Quote (ILLinkPath));
+			return args.ToString ();
+		}
+
+		protected override string GenerateResponseFileCommands ()
```

Comment:
👍

Reply:
Can you please make the response file such that there is one argument/switch per line? 
Basically add newlines at the end of each option? 

This makes it easier to inspect the rsp files manually.

Reply:
Done, thanks!